src/current/cockroachcloud/physical-cluster-replication.md

```diff
@@ -0,0 +1,286 @@
+~~~
+
+Replace:
+
+- `api_secret_key` with your API secret key.
+- `primary_cluster_id` with the cluster ID returned after creating the primary cluster.
+- `standby_cluster_id` with the cluster ID returned after creating the standby cluster.
+
+You can find the cluster IDs in the cluster creation output, or in the URL of the single cluster overview page: `https://cockroachlabs.cloud/cluster/{your_cluster_id}/overview`. The ID will resemble `ad1e8630-729a-40f3-87e4-9f72eb3347a0`.
+
+{{site.data.alerts.callout_info}}
+Once you have started PCR, the standby cluster cannot accept writes and reads, therefore the [Cloud Console]({% link cockroachcloud/cluster-overview-page.md %}) and SQL shell will be unavailable prior to failover.
+{{site.data.alerts.end}}
+
+You will receive a response similar to:
+
+~~~ json
+{
+  "id": "c3d35c84-a4ea-41b3-8452-553c5ded3b85",
+  "status": "STARTING",
+  "primary_cluster_id": "3fabc29e-5ced-48d9-b31e-000000000000",
+  "standby_cluster_id": "f9b1d580-9be3-47f8-ac28-000000000000",
+  "created_at": "2025-05-01T18:57:50.038137Z"
+}
+~~~
+
+- `"id"`: The PCR stream's job ID.
+- `"status"`: The status of the PCR stream. For descriptions, refer to [Status](#status).
+- `"primary_cluster_id"`, `"standby_cluster_id"`: The cluster IDs of the primary and standby clusters.
+- `"created_at"`: The time at which the PCR stream was created.
+
+To start PCR between clusters, CockroachDB {{ site.data.products.cloud }} sets up VPC peering between clusters and validates the connectivity. As a result, it may take around 5 minutes to initialize the PCR job during which the status will be `STARTING`.
+
+### Step 3. Monitor the PCR stream
+
+For monitoring the current status of the PCR stream, send a `GET` request to the `/v1/physical-replication-streams` endpoint along with the ID of the PCR stream:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --request GET "https://cockroachlabs.cloud/api/v1/physical-replication-streams/job_id" --header "Authorization: Bearer api_secret_key" 
+~~~
+
+Replace:
+
+- `api_secret_key` with your API secret key.
+- `job_id` with the PCR job's ID. You can find this in the response from when you created the PCR stream.
+
+This will return a response similar to:
+
+~~~json
+{
+  "id": "c3d35c84-a4ea-41b3-8452-553c5ded3b85",
+  "status": "REPLICATING",
+  "primary_cluster_id": "3fabc29e-5ced-48d9-b31e-000000000000",
+  "standby_cluster_id": "f9b1d580-9be3-47f8-ac28-000000000000",
+  "created_at": "2025-05-01T18:57:50.038137Z",
+  "retained_time": "2025-05-01T19:02:36.462825Z",
+  "replicated_time": "2025-05-01T19:05:25Z",
+  "replication_lag_seconds": 9
+}
+~~~
+
+- `"id"`: The ID of the PCR stream.
+- `"status"`: The status of the PCR stream. For descriptions, refer to [Status](#status).
+- `"primary_cluster_id"`, `"standby_cluster_id"`: The cluster IDs of the primary and standby clusters.
+- `"created_at"`: The time at which the PCR stream was created.
```

Comment:
same q as above re: what exactly are these times, are they SQL timestamps, etc.

Reply:
They're date-time, I've linked to the timestamp docs, also the schema of the response is listed in the Cloud API docs.